desktop/index.md

```diff
@@ -49,6 +49,19 @@ Docker Desktop is available for Mac and Windows. For download information, syste
 
 For information about Docker Desktop licensing, see [Docker Desktop License Agreement](../subscription/index.md#docker-desktop-license-agreement).
 
+## Sign in to Docker Desktop
+
+After you’ve successfully installed and started Docker Desktop, we recommend that you authenticate using the **Sign in/Create ID** option from the Docker menu.
+
+Authenticated users get a higher pull rate limit compared to anonymous users. For example, if you are authenticated, you get 200 pulls per 6 hour period per IP address, compared to 100 pulls per 6 hour period per IP address for anonymous users. For more information, see [Download rate limit](../docker-hub/download-rate-limit.md).
```

Comment:
If you are authenticated, you get those pulls per user, not per IP address.

Reply:
OK, in that case, does it make sense to say "f you are authenticated, you get 200 pulls per 6 hour period, compared to 100 pulls per 6 hour period per IP address for anonymous users."

Reply:
Yes

Reply:
Thanks. Updated